.github/workflows/publish-to-pypi.yml

```diff
@@ -17,6 +17,22 @@ jobs:
       - name: Checkout the repo
         uses: actions/checkout@v2
       
+      # Build JS
```

Comment:
Added JS build step to the publishing workflow. This builds the JS code and copies it to `src/django_components/static/django_components/__generated__`, from which it is then copied into the package via `MANIFEST.in`.

The JS code can then be used in Python / Django as so
```py
static("django_components/__generated__/django_components.min.js")
```